tox.ini

```diff
@@ -0,0 +1,56 @@
+[tox]
+envlist =
+    flake8
+    isort
+    py{27,34,35,36}-dj{111,21,22}
```

Comment:
Same with classy tags, `py27-dj{21,22}` shouldn't be valid envs. Minor thing, though.